ui/widgets/nuxeo-selectivity.js

```diff
@@ -7350,7 +7350,10 @@ typedArrayTags[weakMapTag] = false;
 
     _readonlyChanged() {
       if (this._selectivity) {
-        this._selectivity.setOptions({ readOnly: this.readonly });
+        this._selectivity.setOptions({
+          readOnly: this.readonly,
+          placeholder: this.readonly ? "" : this.placeholder,
```

Comment:
This is inconsistent with other widgets, where the placeholder is visible even on read-only mode (e.g. `nuxeo-input`). Perhaps in some contexts, it might even represent a breaking change? Fixing it on `nuxeo-tag-suggestion` would be less risky, but maybe we should fix this on the element that makes use of `nuxeo-tag-suggestion`?

Reply:
Any progress on this @semisse?

Reply:
This is the ticket I'm currently tackling. We are looking for alternatives to accomplish the same result in Web UI side, however we still need to fix it in `nuxeo-selectivity`.